tests/ClientBuilderTest.php

```diff
@@ -295,7 +247,9 @@ public function testClientBuilderSetsSdkIdentifierAndVersion(): void
      */
     public function testGetClientTogglesCompressionPluginInHttpClient(bool $enabled): void
     {
-        $builder = ClientBuilder::create(['enable_compression' => $enabled, 'dsn' => 'http://public:secret@example.com/sentry/1']);
+        $options = new Options(['dsn' => 'http://public:secret@example.com/sentry/1']);
```

Comment:
Instead of instantiating the `Options` object I would prefer to stick with the plain `array` of options